core/settings/qPref.h

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: GPL-2.0
+#ifndef QPREF_H
+#define QPREF_H
+
+#include <QObject>
+#include "core/pref.h"
+
+class qPref : public QObject {
+	Q_OBJECT
+	Q_ENUMS(cloud_status);
+
+public:
+	qPref(QObject *parent = NULL) : QObject(parent) {};
+	~qPref() {};
```

Comment:
Any reason to overwrite the default-destructor by an inlined version?
(Note: If you do, then the semicolon is not needed.)